debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -62,6 +68,59 @@
+        DEFAULT_VALUE_MAPPERS.put("int", v -> Integer.parseInt(v.substring(2, v.length() - 2)));        // Sample value: ((2147483647))
+        DEFAULT_VALUE_MAPPERS.put("smallint", v -> Short.parseShort(v.substring(2, v.length() - 2)));   // Sample value: ((32767))
+        DEFAULT_VALUE_MAPPERS.put("tinyint", v -> Short.parseShort(v.substring(2, v.length() - 2)));    // Sample value: ((255))
+        DEFAULT_VALUE_MAPPERS.put("bit", v -> v.equals("((1))"));                                       // Either ((1)) or ((0))
+        DEFAULT_VALUE_MAPPERS.put("decimal", v -> new BigDecimal(v.substring(2, v.length() - 2)));      // Sample value: ((100.12345))
+        DEFAULT_VALUE_MAPPERS.put("numeric", v -> new BigDecimal(v.substring(2, v.length() - 2)));      // Sample value: ((100.12345))
+        DEFAULT_VALUE_MAPPERS.put("money", v -> new BigDecimal(v.substring(2, v.length() - 2)));        // Sample value: ((922337203685477.58))
+        DEFAULT_VALUE_MAPPERS.put("smallmoney", v -> new BigDecimal(v.substring(2, v.length() - 2)));   // Sample value: ((214748.3647))
+
+        // Approximate numerics
+        DEFAULT_VALUE_MAPPERS.put("float", v -> Double.parseDouble(v.substring(2, v.length() - 2)));    // Sample value: ((1.2345000000000000e+003))
+        DEFAULT_VALUE_MAPPERS.put("real", v -> Float.parseFloat(v.substring(2, v.length() - 2)));       // Sample value: ((1.2345000000000000e+003))
+
+        // Date and time
+        DEFAULT_VALUE_MAPPERS.put("date", v -> Date.valueOf(v.substring(2, v.length() - 2)));           // Sample value: ('2019-02-03')
+        DEFAULT_VALUE_MAPPERS.put("datetime", v -> Timestamp.valueOf(v.substring(2, v.length() - 2)));  // Sample value: ('2019-01-01 00:00:00.000')
+        DEFAULT_VALUE_MAPPERS.put("datetime2", v -> Timestamp.valueOf(v.substring(2, v.length() - 2))); // Sample value: ('2019-01-01 00:00:00.1234567')
+        DEFAULT_VALUE_MAPPERS.put("datetimeoffset", v -> {                                              // Sample value: ('2019-01-01 00:00:00.1234567+02:00')
+            String rawValue = v.substring(2, v.length() - 2);
+            String datetime = rawValue.substring(0, rawValue.length() - 6);
+            String offset = rawValue.substring(rawValue.length() - 6);
+            int offsetSignum = offset.charAt(0) == '-' ? -1 : 1;
+            int offsetHours = Integer.parseInt(offset.substring(1, 3));
+            int offsetMinutes = Integer.parseInt(offset.substring(4, 6));
+            int minutesOffset = offsetSignum * (offsetHours * 60 + offsetMinutes);
+            return DateTimeOffset.valueOf(Timestamp.valueOf(datetime), minutesOffset);
+        });
+        DEFAULT_VALUE_MAPPERS.put("smalldatetime", v -> Timestamp.valueOf(v.substring(2, v.length() - 2))); // Sample value: ('2019-01-01 00:00:00')
+        DEFAULT_VALUE_MAPPERS.put("time", v -> new Timestamp(Time.valueOf(v.substring(2, v.length() - 2)).toLocalTime().toSecondOfDay() * 1000L));  // Sample value: ('00:00:56')
```

Comment:
https://docs.microsoft.com/en-us/sql/t-sql/data-types/time-transact-sql?view=sql-server-ver15

Nonoseconds are not covered.